agent/requestor/src/main.rs

```diff
@@ -156,7 +179,7 @@ async fn main() -> anyhow::Result<()> {
     let subscription_id = api.market.subscribe(&my_demand).await?;
     log::info!("\n\n DEMAND SUBSCRIBED: {}", subscription_id);
 
-    let shutdown = shutdown_handler(
+    let (future_shutdown, app_abort_handle) = shutdown_handler(
```

Comment:
a common name for future
```suggestion
    let (shutdown_fut, app_abort_handle) = shutdown_handler(
```